### DISCUSSION TITLE
使用2.90 最后下载仍然是填充无效

### DISCUSSION THREAD

**Discussion by SherlockTan** [nilaoda/N_m3u8DL-CLI#302] [General]
N_m3u8DL-CLI v2.9.0 20201206...
Speed: 142.97 KB / s
Progress: waiting

11:19:14.955 文件名称：voddrm.token.eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9~eyJ0eXBlIjoiRHJtVG9rZW4iLCJhcHBJZCI6MTI1NjUxNzQ11:19:14.968 存储路径：C:\Users\tanwanying\Downloads\N_m3u8DL-CLI_v2.9.0_with_ffmpeg_and_SimpleG\Downloads
11:19:14.983 开始解析
11:19:15.074 获取m3u8内容
11:19:15.703 解析m3u8内容
11:19:15.706 获取m3u8 key...
11:19:16.111 写出meta.json
11:19:16.349 文件时长：10m22s
11:19:16.352 总分片：60, 已选择分片：60
11:19:16.388 开始下载文件
11:19:16.403 下载首分片...
11:19:17.677 填充无效，无法被移除。

我下载时仍旧是填充无效，提供一些数据供参考，作者很强，给作者比心！

**Comment by nilaoda** [comment]
气球云吗？等有空的时候我会重新搞一下的
